index.js

```diff
@@ -398,6 +409,30 @@ function fastifyView (fastify, opts, next) {
   } else {
     next()
   }
+
+  function withLayout (render) {
+    if (layoutFileName) {
```

Comment:
How is this option going to affect other engines?

Reply:
Currently it would only affect `handlebars`. This is because some engines already support their own layouts (e.g: `pug`).

It is also possible to add support to other engines that don't already support layouts by applying them the `withLayout` decorator (see [here](https://github.com/fastify/point-of-view/pull/136/files#diff-168726dbe96b3ce427e7fedce31bb0bcR44))

Reply:
would you mind throwing if `layout` is present and the engine is not `'handlebars'`?

Reply:
No problem. Please see my new commits.